### PR TITLE
Migrated BinaryClassification_SentimentAnalysis F# sample to Dynamic API

### PR DESCRIPTION
**Pull request by mariuszwojcik**
Migration done by following C# sample.
I have created simple `Pipeline` module to make some operations on pipeline more F# friendly. This allows to **pipe** operations for creating pipeline and training model:

```fsharp
    let model = 
        env
        |> Pipeline.textTransform "Text" "Features"
        |> Pipeline.append (LinearClassificationTrainer(env, LinearClassificationTrainer.Arguments(), "Features", "Label"))
        |> Pipeline.fit trainingDataView
```
I plan to *enhance* this module as I continue with migration of other projects.

### CODE REVIEW

**Comment by CESARDELATORRE** [comment]
@mariuszwojcik Thanks! 👍 - @dsyme Don, when possible, can you review this PR of this F# sample?

Thanks for your contribution Marius! 👍

**Comment by kunjee17** [comment]
@CESARDELATORRE I guess migrating to dynamic module is already in progress. If @dsyme clears this one. I continue to move forward from work done by @mariuszwojcik so we don't do same thing twice.

**Comment by mariuszwojcik** [comment]
Hi Kunjan,

Did you already started migrating F# samples to dynamic API? Which ones? I already started on MulticlassClassification_Iris. Please let me  know which samples you will work on.

Mariusz

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

File: samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/BinaryClassification_SentimentAnalysis.fsproj [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
-    <Folder Include="datasets\" />
-    <None Include="..\..\..\..\datasets\sentiment-imdb-train.txt" Link="datasets\sentiment-imdb-train.txt" />
-    <None Include="..\..\..\..\datasets\sentiment-yelp-test.txt" Link="datasets\sentiment-yelp-test.txt" />
   </ItemGroup>
 
+  <ItemGroup />
+
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="$(MicrosoftMLVersion)" />
+    <PackageReference Include="Microsoft.ML" Version="0.6.0" />
```
Comment (dsyme) [inline samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/BinaryClassification_SentimentAnalysis.fsproj]: Is there any reason not to continue to use `$(MicrosoftMLVersion)`?

File: samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/Program.fs [hunk 2]
```diff
@@ -1,123 +1,141 @@
...
-    { [<Column("0")>]
-      SentimentText: string
-      [<Column("1", name="Label")>]
-      Sentiment: float32 }
 
-[<CLIMutable>]
-type SentimentPrediction =
-    { [<ColumnName("PredictedLabel")>]
-      Sentiment: bool }
+type [<CLIMutable>] SentimentIssue = { Label : bool; Text : string }
```
Comment (dsyme) [inline samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/Program.fs]: I think beginners prefer to see plenty of whitespace, so format these over multiple lines with a line whitespace in between.  Also add a `///` comment to both

**Comment by dsyme** [comment]
Just a couple of small comments above.

What automated testting do we have in place for the samples? (if any?) 

thanks

File: samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/BinaryClassification_SentimentAnalysis.fsproj (hunk 1 above)
Comment (CESARDELATORRE) [reply samples/fsharp/getting-started/BinaryClassification_SentimentAnalysis/BinaryClassification_SentimentAnalysis.fsproj]: We must continue using $(MicrosoftMLVersion)
Only as a temporal case, if using a nightly build like 0.7-Preview in MyGet we can use a fixed nuget version.

**Comment by mariuszwojcik** [comment]
@kunjee17 thanks, I made suggested changes already. My plan is to migrate rest of the samples to F#. Lets coordinate our efforts using issue #83 - simply add comment on which sample you are working before starting

**Pull request merged**